package.json

```diff
@@ -32,19 +32,21 @@
     "babel-plugin-transform-runtime": "^6.23.0",
     "babel-preset-env": "^1.6.0",
     "bluebird": "^3.5.1",
-    "ethereum-claims-registry": "^0.0.4",
     "ganache-cli": "^6.1.0",
     "jest": "^22.4.0",
     "regenerator-runtime": "^0.11.1",
     "sinon": "^4.0.1",
     "standard": "^11.0.0",
-    "web3": "0.20.6"
+    "web3": "^1.0.0-beta.34"
```

Comment:
Why use the beta of web3 1.0.0?

Reply:
It's the default when you install web3. The `0.2` is not supported anymore and throws error when you try to install it.

Reply:
Ok got it. Weird that they call it 1.0 beta when 0.2 is no longer supported :\